src/assets/styles/_animation.scss

```diff
@@ -41,7 +41,11 @@
 }
 
 :root {
-  --calcite-animation-timing: 300ms;
+  --animation-timing-factor: var(--calcite-animation-timing-factor, 1);
```

Comment:
Per our conversation w/ @macandcheese, all root internal CSS props should be namespaced w/ `--calcite-internal-<prop>`. Please rename the following:

* `--animation-timing-factor` -> `--calcite-internal-animation-timing-factor`
* `--calcite-animation-timing-quick` -> `--calcite-internal-animation-timing-quick`
* `--calcite-animation-timing-medium` -> `--calcite-internal-animation-timing-medium`
* `--calcite-animation-timing-slow` -> `--calcite-internal-animation-timing-slow`

Note: `--calcite-animation-timing` stays the same as it is public.